cms/djangoapps/contentstore/views/course.py

```diff
@@ -1502,8 +1578,28 @@ def get_or_create_content_group_configuration(course):
                 groups=[],
                 scheme_id='cohort'
             )
+            return content_group_configuration.to_json()
+
+        content_group_configuration = GroupConfiguration.update_content_group_usage_info(
+            store,
+            course,
+            content_group_configuration
+        )
         return content_group_configuration
 
+    @staticmethod
+    def remove_content_group_from_configuration(configuration, group_id):
```

Comment:
I'd call this ``remove_group_from_configuration`` since it doesn't have any logic specific to content groups.